templates/bake/config/snapshot.twig

```diff
@@ -40,12 +40,12 @@ class {{ name }} extends AbstractMigration
      */
     public function up()
     {
-{% element 'Migrations.create-tables' {
+{{~ element('Migrations.create-tables', {
```

Comment:
Should this be `{{-`?

Reply:
no , then it removes spaces + new lines from the beginning
see the result after replacing with `{{-`

```php
    public function up()
    {        $this->table('posts')
            ->addColumn('title', 'string', [
                'default' => null,
                'limit' => 1000,
                'null' => false,
            ])
```

Reply:
in my test in local , current code works well